src/Sql/dbo/Dirt/Stored Procedures/OrganizationApplication_Create.sql

```diff
@@ -3,7 +3,8 @@ CREATE PROCEDURE [dbo].[OrganizationApplication_Create]
     @OrganizationId UNIQUEIDENTIFIER,
     @Applications NVARCHAR(MAX),
     @CreationDate DATETIME2(7),
-    @RevisionDate DATETIME2(7)
+    @RevisionDate DATETIME2(7),
+    @ReportKey VARCHAR(MAX)
```

Comment:
Question: Since this is not concerning reports but applications, Should this not be "ApplicationsKey" or "ApplicationKey"? (I'm not sure what's stored in the encrypted data blob).

(This applies to all other files concerning organizationApplication)